segmentio/analytics-kotlin
# Add amplitude session plugin sample

Test Plan:

- Verify sessionID starts as -1
    - Verify when the application starts, sessionID is -1 by placing a breakpoint in startTimer()
    
- Verify timer fires
    - Verify timer fires after 5 min (+/- 1 second)
    - Verify sessionID changes after fire
    - Verify amplitude integration payload session_id is updated

- Verify sessionID once set is 13 digits
    - Set a breakpoint at line 81 (in startTimer()) to verify the sessionID has changed and is 13 digits. 


<img width="1904" alt="Screen Shot 2021-06-03 at 2 15 33 PM" src="https://user-images.githubusercontent.com/917994/120715280-40ab7f00-c479-11eb-8d1d-1f243c65c281.png">
<img width="1904" alt="Screen Shot 2021-06-03 at 2 15 44 PM" src="https://user-images.githubusercontent.com/917994/120715291-43a66f80-c479-11eb-9c2c-12015eda0b0e.png">
<img width="1904" alt="Screen Shot 2021-06-03 at 2 16 13 PM" src="https://user-images.githubusercontent.com/917994/120715295-44d79c80-c479-11eb-8fec-ca0152ed4a45.png">
<img width="1904" alt="Screen Shot 2021-06-03 at 2 17 43 PM" src="https://user-images.githubusercontent.com/917994/120715300-4608c980-c479-11eb-99c1-94d1f8b64b4a.png">
<img width="1904" alt="Screen Shot 2021-06-03 at 2 19 04 PM" src="https://user-images.githubusercontent.com/917994/120715302-46a16000-c479-11eb-8e0d-724f399df8c3.png">
<img width="1904" alt="Screen Shot 2021-06-03 at 2 19 19 PM" src="https://user-images.githubusercontent.com/917994/120715305-4739f680-c479-11eb-8cdb-3fff047ff91c.png">
